content/docs/next-release/operator.md

```diff
@@ -881,6 +881,44 @@ kubectl delete jaeger simplest
 Deleting the instance will not remove the data from any permanent storage used with this instance. Data from in-memory instances, however, will be lost.
 {{< /info >}}
 
+# Tracing the operator
+
+Starting from version 1.16.0, the Jaeger Operator is able to generate spans related to its own operations. To take advantage of that, the `operator.yaml` has to be configured to enable tracing by setting the flag `--tracing-enabled=true` to the `args` of the container and to add a Jaeger Agent as sidecar to the pod. Here's an excerpt from an `operator.yaml` that has tracing enabled:
+
+```yaml
+...
+        # .Spec.Template.Spec.Containers[0].Args
+        args: ["start", "--tracing-enabled=true"]
+...
+      # add as a second container to .Spec.Template.Spec.Containers
+      - name: jaeger-agent
+        image: jaegertracing/jaeger-agent:1.15.1
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        args:
+        - --reporter.grpc.host-port=dns:///jaeger-collector-headless.$(POD_NAMESPACE).svc.cluster.local:14250
+        ports:
+        - containerPort: 6831
+          name: jg-compact-trft
+          protocol: UDP
+```
+
+Note that you must also manually provision the Jaeger instance. You can do this after the Jaeger Operator has been initialized. The Jaeger Agent will keep the Operator spans in the internal buffer until it makes a connection to the Jaeger instance. The following Jaeger CR can be used to provision a Jaeger instance suitable for non-production purposes:
```

Comment:
The excerpt above assumes the jaeger instance is in the same namespace as the operator - but this para or following CR doesn't make that clear.

Reply:
Added the clarification in the paragraph before the snippet.